File: Q01.js
/**
 * Coding Interview Question 01
 */

for (var i = 0; i < 3; i++) {
  setTimeout(() => {
    console.log(`With var = ${i}`);
  }, 1000 + i);
}

// output: 3 3 3

for (let i = 0; i < 3; i++) {
  setTimeout(() => {
    console.log(`With let = ${i}`);
  }, 1000 + i);
}

// output: 0 1 2

File: JAVA/Q01.java
package JAVA;

// ######################################################
// Interview Question #01
// Write a program to find the age of a person in years.
// #####################################################$

import java.time.LocalDate;
import java.time.Period;

public class Q01 {

	public static void main(String[] args) {
		LocalDate today = LocalDate.now();
		LocalDate birthDate = LocalDate.of(1967, 1, 11);
		Period period = Period.between(birthDate, today);
		System.out.println("Age: " + period.getYears());
	}
}
